src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -1014,4 +1015,53 @@ public static String getCatchErrorResult(BlockStartNode startNode) {
         }
         return null;
     }
-}
\ No newline at end of file
+
+    /**
+     * Checks to see if event can be sent to client
+     * @param event - the event to check
+     * @return true if event is can be sent to client
+     */
+    public static boolean shouldSendEvent(String eventName) {
```

Comment:
The description and the function name don't appear to have the same meaning.

Reply:
Good point, updated to always be shouldSendEvent